group_project_v2/app_config.py

```diff
@@ -32,7 +32,7 @@
 
 ENTRYPOINTS = [
     "{entrypoint} = {class_location}".format(entrypoint=entrypoint, class_location=class_location)
-    for entrypoint, class_location in BLOCKS.iteritems()
+    for entrypoint, class_location in BLOCKS.items()
 ]
 
-PROGRESS_DETACHED_CATEGORIES = BLOCKS.keys()
+PROGRESS_DETACHED_CATEGORIES = list(BLOCKS.keys())
```

Comment:
Can't use six here, because this code gets imported during pip install (via setup.py via requirements/base.txt which requires `-e .`).